relm4/src/channel/component.rs

```diff
@@ -145,6 +145,7 @@ macro_rules! sender_impl {
                 shutdown: ShutdownReceiver,
             ) -> Self {
                 Self {
+                    #[allow(clippy::all)]
                     shared: Arc::new(ComponentSenderInner {
```

Comment:
Why is this necessary? I don't get a warning here and even if clippy complained, it doesn't make sense to ignore all clippy lints here.